tests/wait_for_postgres.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+PG_ISREADY=/usr/lib/postgresql/9.6/bin/pg_isready
+echo "checking postgres"
```

Comment:
This echo is unnecessary in the output that shows up since `Sleeping while postgres starts up` immediately precedes it:
```
Creating the docker network
5bae4bd9e1cb4713e20cf2cac51bbc7a2c72e630b493a197fc66e809f20e7b07
Starting postgres
8f589328be06b0675472ba7b7ddfa61cfeb978116a091b0db353352737bc0a32
Sleeping while postgres starts up
checking postgres
pgbedrock_postgres:5432 - no response
waiting on postgres
pgbedrock_postgres:5432 - no response
waiting on postgres
pgbedrock_postgres:5432 - no response
waiting on postgres
pgbedrock_postgres:5432 - no response
waiting on postgres
pgbedrock_postgres:5432 - accepting connections
Running pytest with Python 2.7
```